README.md

```diff
@@ -69,17 +69,45 @@ production)
 Use the `--help` command-line flag to get a more detailed description of each
 tool's options.
 
-### Connection setup
+### Configuring connections
 
 The first step in establishing a QUIC connection using quiche is creating a
-configuration object:
+[`Config`] object:
 
 ```rust
-let config = quiche::Config::new(quiche::PROTOCOL_VERSION)?;
+let mut config = quiche::Config::new(quiche::PROTOCOL_VERSION)?;
+config.set_application_protos(&[b"h3"]);
```

Comment:
This overview is supposed to be somewhat generic, so using the h3 ALPN here makes it sound like everyone has to use it regardless of what protocol runs on top of QUIC. Maybe change it to `example-proto` or something?

Reply:
yep